apis/elasticloadbalancing/v1alpha1/elb_types.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+)
+
+// Tag defines a key value pair that can be attached to an ELB
+type Tag struct {
+
+	// The key of the tag.
+	Key string `json:"key"`
+
+	// The value of the tag.
+	// +optional
+	Value *string `json:"value,omitempty"`
+}
+
+// Listener represents the port binding(s) between the ELB and EC2 instances.
+type Listener struct {
+
+	// The port on which the instance is listening.
+	InstancePort int64 `json:"instancePort"`
+
+	// The protocol to use for routing traffic to instances: HTTP, HTTPS, TCP, or
+	// SSL.
+	// +optional
+	InstanceProtocol *string `json:"instanceProtocol,omitempty"`
+
+	// The port on which the load balancer is listening.
+	LoadBalancerPort int64 `json:"loadBalancerPort"`
+
+	// The load balancer transport protocol to use for routing: HTTP, HTTPS, TCP,
+	// or SSL.
+	Protocol string `json:"protocol"`
+
+	// The Amazon Resource Name (ARN) of the server certificate.
```

Comment:
Can we change this comment to something like: `Sets the certificate that terminates the specified listener's SSL connections. The specified certificate replaces any prior certificate that was used on the same load balancer and port.`

Reply:
I think this statement is more for defining an action rather an entity/field. I haven't changed the comment for now.

Reply:
Fair. Got it.